### DISCUSSION TITLE
No green 'expected end of descent/climb' arc on ND

### DISCUSSION THREAD

**Discussion by rtulaza** [magknight/787aviatorsTracker#551] [Completed things] [answered]
**Describe the bug**
When climbing or descending, the green arc that indicates expected level-off distance is not shown. This is very troublesome as it is hard to choose proper V/S without this visual indication. Especially since VNAV is not always working as expected (I am not complaining about this fact, it simply will improve over time) and user has to switch to manual V/S mode.

**Expected behavior**
I think adding this arc will vastly help operate the aircraft while work on other automated systems is still being done.

**Desktop (please complete the following information):**
 - Windows 10
 - X-Plane version 11.41
 - Version 1.4.0

**Comment by aeroniemi** [comment]
We're not able to add it to the ND as we aren't able to edit the content of the default display. It will, however, be included with ND2 in the CDU update

**Reply by aeroniemi** [reply]
The arc is present on the minimap now

**Comment by aeroniemi** [comment, marked as answer]
it's on ND2 on the cygnus branch
